src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -189,137 +189,133 @@ bool OutputStateMachineEngine::ActionPassThroughString(const std::wstring_view /
+        _dispatch->AnnounceCodeStructure(2);
         break;
     case EscActionCodes::ACS_AnsiLevel3:
-        success = _dispatch->AnnounceCodeStructure(3);
+        _dispatch->AnnounceCodeStructure(3);
         break;
     case EscActionCodes::DECDHL_DoubleHeightLineTop:
-        success = _dispatch->SetLineRendition(LineRendition::DoubleHeightTop);
+        _dispatch->SetLineRendition(LineRendition::DoubleHeightTop);
         break;
     case EscActionCodes::DECDHL_DoubleHeightLineBottom:
-        success = _dispatch->SetLineRendition(LineRendition::DoubleHeightBottom);
+        _dispatch->SetLineRendition(LineRendition::DoubleHeightBottom);
         break;
     case EscActionCodes::DECSWL_SingleWidthLine:
-        success = _dispatch->SetLineRendition(LineRendition::SingleWidth);
+        _dispatch->SetLineRendition(LineRendition::SingleWidth);
         break;
     case EscActionCodes::DECDWL_DoubleWidthLine:
-        success = _dispatch->SetLineRendition(LineRendition::DoubleWidth);
+        _dispatch->SetLineRendition(LineRendition::DoubleWidth);
         break;
     case EscActionCodes::DECALN_ScreenAlignmentPattern:
-        success = _dispatch->ScreenAlignmentPattern();
+        _dispatch->ScreenAlignmentPattern();
         break;
     default:
         const auto commandChar = id[0];
         const auto commandParameter = id.SubSequence(1);
         switch (commandChar)
         {
         case '%':
-            success = _dispatch->DesignateCodingSystem(commandParameter);
+            _dispatch->DesignateCodingSystem(commandParameter);
             break;
         case '(':
-            success = _dispatch->Designate94Charset(0, commandParameter);
+            _dispatch->Designate94Charset(0, commandParameter);
             break;
         case ')':
-            success = _dispatch->Designate94Charset(1, commandParameter);
+            _dispatch->Designate94Charset(1, commandParameter);
             break;
         case '*':
-            success = _dispatch->Designate94Charset(2, commandParameter);
+            _dispatch->Designate94Charset(2, commandParameter);
             break;
         case '+':
-            success = _dispatch->Designate94Charset(3, commandParameter);
+            _dispatch->Designate94Charset(3, commandParameter);
             break;
         case '-':
-            success = _dispatch->Designate96Charset(1, commandParameter);
+            _dispatch->Designate96Charset(1, commandParameter);
             break;
         case '.':
-            success = _dispatch->Designate96Charset(2, commandParameter);
+            _dispatch->Designate96Charset(2, commandParameter);
             break;
         case '/':
-            success = _dispatch->Designate96Charset(3, commandParameter);
+            _dispatch->Designate96Charset(3, commandParameter);
             break;
         default:
             // If no functions to call, overall dispatch was a failure.
-            success = false;
             break;

```

Comment:
This is similar to the case I noted in `AdaptDispatch`. This comment seems a bit pointless without the `success = false` and I would suggest nuking it. There are a couple more of these in this file.